server/src/main/java/org/apache/druid/catalog/model/table/ExternalTableDefn.java

```diff
@@ -99,6 +98,8 @@ private static List<PropertyDefn<?>> addFormatProperties(
     )
     {
       List<PropertyDefn<?>> toAdd = new ArrayList<>();
+      PropertyDefn<?> formatProp = new ModelProperties.StringPropertyDefn(FORMAT_PROPERTY, PropertyAttributes.SQL_FN_PARAM);
```

Comment:
question: Is this property key being set with a value here? It appears no. I assume that the user must supply a value for this property based on other checks made in this class. Where is the value for this property set? And if not set, explicitly by the user, could this property creation here with no value interfere with validation checks, and cause an external table that a user hasnt given a format property value for to be seen as being valid because of the existence of the property but with no value?